docs/root/configuration/other_features/reverse_tunnel.rst

```diff
@@ -54,7 +54,7 @@ The initiator Envoy (downstream) requires the following configuration components
 Downstream socket interface
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. literalinclude:: /configs/reverse_connection/initiator-envoy.yaml
+.. literalinclude:: ../../../../configs/reverse_connection/initiator-envoy.yaml
```

Comment:
for ref you _can_ use "absolute" refs - but either works - we dont have a great story around docs resources (and refs)